src/main/resources/scripts/ubuntuInstallDockerScript.sh

```diff
@@ -0,0 +1,6 @@
+# Install Docker
+sudo apt-get install -y apt-transport-https ca-certificates curl software-properties-common
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+sudo apt-get -y update
+sudo apt-get install -y docker-ce
```

Comment:
For docker, do we need to add jenkins account to docker group so agent can use docker?